pkg/apiserver/server.go

```diff
@@ -102,7 +102,7 @@ func newDashboardRouter(r *gin.Engine, ui http.FileSystem) {
 		c.FileFromFS(c.Request.URL.Path, ui)
 	}
 
-	r.GET("dashboard/*any", func(c *gin.Context) {
+	r.GET("/dashboard/*any", func(c *gin.Context) {
```

Comment:
Could this issue cause any anomalies or API errors?

Reply:
No, this is just a unified change, prepend a slash to all defined routes. Add it or not won't influence the real effect.